server/src/test/java/io/spine/examples/shareaware/server/watchlist/WatchlistTest.java

```diff
@@ -48,37 +49,71 @@ protected BoundedContextBuilder contextBuilder() {
         return TradingContext.newBuilder();
     }
 
-    @Test
-    @DisplayName("allow the creation and emit the `WatchlistCreated` event")
-    void event() {
-        CreateWatchlist command = generateCommand();
-        context().receivesCommand(command);
-        WatchlistCreated expected = WatchlistCreated
-                .newBuilder()
-                .setOwner(command.getUser())
-                .setWatchlist(command.getWatchlist())
-                .setName(command.getName())
-                .vBuild();
-        EventSubject assertEvents = context()
-                .assertEvents()
-                .withType(WatchlistCreated.class);
-        Watchlist expectedState = Watchlist
-                .newBuilder()
-                .setId(command.getWatchlist())
-                .setName(command.getName())
-                .vBuild();
+    @Nested
+    @DisplayName("allow the creation ")
+    class Creation {
 
-        assertEvents.hasSize(1);
-        context().assertState(command.getWatchlist(), expectedState);
-        context().assertEvent(expected);
+        @Test
+        @DisplayName("and emit the `WatchlistCreated` event")
+        void event() {
+            CreateWatchlist command = createWatchlist();
+            context().receivesCommand(command);
+            WatchlistCreated expected = WatchlistCreated
```

Comment:
In future PRs, please consider extracting the creation of Proto messages into methods, and moving them to the corresponding test environment.

The idea is to have less boilerplate in tests, so that the code would read better.